<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartDetailDlg</class>
 <widget class="QDialog" name="CartDetailDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { 
    border: 1px solid rgb(41, 57, 85);
    background: white;
    selection-background-color: darkgray;
    font-size: 16px ;
}

QLineEdit:hover {
	border: 2px solid blue;
}

QLineEdit[echoMode=&quot;2&quot;] {
      lineedit-password-character: 42;
}

QPushButton{
	border-radius:6px;
	background-color: #1390C5;
	min-width:80px;
}

QPushButton:pressed {
	background-color: #105D7E;
}

QPushButton:hover {
	background-color: #2BA0F3;
}

color: rgb(211, 211, 211);

</string>
  </property>
  <widget class="QLineEdit" name="priceEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="writerEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>付款</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scoreEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pressEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>书名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>价格:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>出版社:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>作者:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>评分:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>简介:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>CartDetailDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
